lib/index.js

```diff
@@ -53,12 +53,13 @@ class Rooftop {
       }
 
       if (transformFn === true) transformFn = transform
-      if (transformFn === false) transformFn = (x) => x
+      else if (transformFn === false) transformFn = (x) => x
```

Comment:
**note:** the ternary is redundant in this case because we only need to check if the value is `false`. The custom fn assignment is done in the if/else block prior to this one.